### PR TITLE
Added steps on how to find values for Azure health source

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \Instructions for Azure health source
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [ ] Code owner review.
- [X] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/service-reliability-management/monitored-service/health-source/azurelogs.md [hunk 1]
```diff
@@ -69,6 +69,16 @@ The new group and metric are created. The query specifications and mapping setti
    * Total 
    * Count
 
+:::note
+
+You will be able to find the values necessary for each query using the Azure UI. Generally you have to follow these steps:
```
Comment (douglas-j-bothwell) [inline docs/service-reliability-management/monitored-service/health-source/azurelogs.md:74]:
```suggestion
You can find the values necessary for each query using the Azure UI. Generally you need to follow these steps:
```

File: docs/service-reliability-management/monitored-service/health-source/azurelogs.md [hunk 2]
```diff
@@ -69,6 +69,16 @@ The new group and metric are created. The query specifications and mapping setti
...
+
+1. Select the [Azure Metric](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/analyze-metrics) you would like to analyze.
```
Comment (douglas-j-bothwell) [inline docs/service-reliability-management/monitored-service/health-source/azurelogs.md:76]:
```suggestion
1. Select the [Azure Metric](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/analyze-metrics) you want to analyze.
```

File: docs/service-reliability-management/monitored-service/health-source/azurelogs.md [hunk 3]
```diff
@@ -69,6 +69,16 @@ The new group and metric are created. The query specifications and mapping setti
...
+2. Click on the [`Share` option](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/analyze-metrics#share-your-charts) in the dashboard and select `Copy Link`.
+3. Input your URL into a [URL decoder](https://meyerweb.com/eric/tools/dencoder/). This should result in you seeing a JSON payload.
+4. In the JSON Payload you decode you should be able to find the **resource ID**, **metric namespace**, and **metric name**.
```
Comment (douglas-j-bothwell) [inline docs/service-reliability-management/monitored-service/health-source/azurelogs.md:79]:
```suggestion
4. In the JSON payload you decoded, you should be able to find the **resource ID**, **metric namespace**, and **metric name**. 
```

**Review by douglas-j-bothwell — DISMISSED** [review]
A few optional sugs otherwise LGTM

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**